packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCardPanel.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+
+import { Panel, PanelType } from 'office-ui-fabric-react/lib/Panel';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+import { AddCard } from './AddCard/AddCard';
+import { getStyles } from './AddCardPanel.styles';
+import { IAddCardPanelProps, IAddCardPanelState, IAddCardPanelStyles } from './AddCardPanel.types';
+import { IDGLCard } from '../../../index';
```

Comment:
is this correct? or should it be fabric?

Reply:
No, internal representation so correct. Although I missed what you pointed in the example file.